test/integration-tests/longCallSpreadExpireItm.test.ts

```diff
@@ -271,7 +271,7 @@ contract('Long Call Spread Option expires Itm flow', ([accountOwner1, nakedBuyer
 
       // Check that we start at a valid state
       const vaultBefore = await controllerProxy.getVault(accountOwner1, vaultCounter1)
-      const vaultStateBefore = await calculator.getExcessCollateral(vaultBefore)
+      const vaultStateBefore = await calculator.getExcessCollateral(vaultBefore[0], vaultBefore[1])
```

Comment:
Can u re-write this as
```javascript
const [vaultBefore, vaultType] = await controllerProxy.getVault(accountOwner1, vaultCounter1)
const vaultStateBefore = await calculator.getExcessCollateral(vaultStateBefore, vaultType)
```

or 

```javascript
const [vaultBefore, vaultType] = await controllerProxy.getVault(accountOwner1, vaultCounter1)
const [amount, isExcess] = await calculator.getExcessCollateral(vaultStateBefore, vaultType)
```

Using [0] and [1] here on both result of `getVault` (which represent collateral & type) and `getExcessCollateral` (which represent `amount`, `isExcess`) is very confusing

Reply:
This need a refactor in different integration files, will do it later in another PR!

Reply:
can u make an issue for it and make sure it happen? don't want to increase our technical debt 😸

Reply:
mmm not sure it is a technical debt even, as it is just another way or writing! but yeah will open an issue.